.env.production

```
@@ -0,0 +1,4 @@
+# Production Environment
+VITE_BASE_URL=https://paperlyte.com
+VITE_SEO_KEYWORDS="note-taking app, simple notes, fast notes, offline notes, tag-based organization, distraction-free writing, minimalist notes"
+VITE_OG_IMAGE=https://paperlyte.com/og-image.png
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_OG_IMAGE key should go before the VITE_SEO_KEYWORDS key

(UnorderedKey)